includes/Classifai/Providers/OpenAI/ChatGPT.php

```diff
@@ -1018,4 +1048,31 @@ public function get_content( int $post_id = 0, int $return_length = 0, bool $use
 		return apply_filters( 'classifai_chatgpt_content', $content, $post_id );
 	}
 
+	/**
+	 * This function should return language for prompt.
+	 *
+	 * @since x.x.x
+	 * @param string $language_code Language code.
+	 */
+	private function get_language_for_prompt( string $language_code ): string {
+		$language = '';
+
+		if ( 'en' === $language_code ) {
+			return $language;
+		}
+
+		$translations = \get_site_transient( 'available_translations' );
+		if ( empty( $translations ) ) {
+			require_once ABSPATH . 'wp-admin/includes/translation-install.php';
+			$translations = wp_get_available_translations();
+		}
+
+		// Use English name for language if available.
+		if ( array_key_exists( $language_code, $translations ) ) {
+			$language_english_name = $translations[ $language_code ]['english_name'] ?? '';
+			$language              = " in $language_english_name  language";
```

Comment:
Seems like extra space here:
```suggestion
			$language              = " in $language_english_name language";
```